src/lib/utils/validatesInputs.tsx

```diff
@@ -45,6 +45,21 @@ export const validateAddressInput = (value: string, callback: React.Dispatch<Err
   callback(response);
 };
 
+const MIN_ALLOWED_VALIDATORS = 1;
+const MAX_ALLOWED_VALIDATORS = 100;
```

Comment:
This kind of functionality should be somewhere in lib/validators/...
Also tsx should be ts.
Also filename itself does not have good naming

Reply:
removed this unnecessary part